code/script.js

```diff
@@ -1,9 +1,86 @@
-// A function that adds and remove the class "active" on the section you click on.
-// We haven't really talked about ´this´ yet, but we will... ;)
-// console.log(this) to see how it works
-function toggle() {
-  this.classList.toggle("active")
+
+const question = document.querySelectorAll(".question")  /* querySelectorAll returns a Nodelist object collection 
+- in other words all element with exact class name/names*/
+let i; /* declaring i */
+
+for (i = 0; i < question.length; i++) {  /* here is for loop starts by assigning an initial expression (value) of 0 to the "i" variable , 
+  "i < question.length" is a conditional expresion that specifies that "i" cant go over the length of "question" (array), 
+  and i++ (increment operator ++) means that each time loop is activated 1 will be added to the "i" variable */
+  
```

Comment:
Sounds great but I do not really understand this one. Maybe you can show me one day. Im intrigued!

Reply:
JavaScript Loops - YouTube -https://www.youtube.com/watch?v=s9wW2PpJsmQ
This is a cool channel to get professional explanation